manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18

```

```suggestion
PackageVersion: 1.1.18
```
This should match control panel. If it does not, `AppsAndFeaturesEntries` are required in the manifest